.ci/Jenkinsfile_cloud

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env groovy
 library 'kibana-pipeline-library'
 def loadPipeline
-def mavenImage =  'maven:3.6.3-openjdk-8-slim'
+def MAVEN_IMAGE =  'maven:3.6.3-openjdk-8-slim'
+def MAX_TEST_RUNS = 5
+def MAX_DEPLOYMENTS = 3
```

Comment:
With this PR we can run multiple simulations against a single deployment in multiple times:
repeat each simulation up to 5 times in a row for a the same deployment;
spin up to 3 deployments and run simulations in parallel

Reply:
Can we increase `MAX_TEST_RUNS` to 10? I'd like to be able to run a total of 30 simulations runs in a single job for APM load testing.